website/docs/cli/commands/describe/describe-affected.md

```diff
@@ -0,0 +1,174 @@
+---
+title: atmos describe affected
+sidebar_label: affected
+sidebar_class_name: command
+id: affected
+description: This command produces a list of the affected Atmos components and stacks given two Git commits.
+---
+
+:::note Purpose
+Use this command to show a list of the affected Atmos components and stacks given two Git commits.
+:::
+
+## Description
+
+The command uses two different Git commits to produce a list of affected Atmos components and stacks.
+
+For the first commit, the command assumes that the current repo root is a Git checkout. An error will be thrown if the current repo is not a Git
+repository (`.git` folder does not exist or is configured incorrectly).
+
+The second commit is specified on the command line by using
+the `--ref` ([Git References](https://git-scm.com/book/en/v2/Git-Internals-Git-References)) or `--sha` (commit SHA) flags.
+
+Either `--ref` or `--sha` should be used. If both flags are provided at the same time, the command will first clone the remote branch pointed to by
+the `--ref` flag and then checkout the Git commit pointed to by the `--sha` flag (`--sha` flag overrides `--ref` flag).
+
+If the flags are not provided, the `ref` will be set automatically to the reference to the default branch (e.g. `main`) and the commit SHA will point
+to the `HEAD` of the branch.
+
+The command works by:
+
+- Cloning the branch (`--ref`) or checking out the commit (`--sha`) of the remote target branch
+- Deep merging all stack configurations for both the current working branch and the target branch
+- Looking for changes in the component directories
+- Comparing each section of the stack configuration looking for differences
+- Output a JSON or YAML document consisting of a list of affected components and stacks and what caused it to be affected
+
```

Comment:
@aknysh if both the component and stacks are affected, which will be output as the `affected` entity?